ApplicationCode/ProjectDataModel/RimGeoMechCase.h

```diff
@@ -163,6 +163,8 @@ class RimGeoMechCase : public RimCase
     caf::PdmField<QString>                               m_initialPermeabilityResultAddress;
     caf::PdmField<double>                                m_permeabilityExponent;
 
+    caf::PdmField<double> m_waterDensityShearSlipIndicator;
```

Comment:
Will load from project file be propagated to Rig data structures?

Reply:
Ah, good catch. Fixed in separate commit.